src/components/LazyLog/README.md

```diff
@@ -1,11 +1,23 @@
-Normal log viewing:
+**Normal log viewing:**
+
+Case Insensitive Search
 
 ```js
 const url = 'https://gist.githubusercontent.com/helfi92/96d4444aa0ed46c5f9060a789d316100/raw/ba0d30a9877ea5cc23c7afcd44505dbc2bab1538/typical-live_backing.log';
 
 <div style={{ height: 500, width: 902 }}>
-  <LazyLog extraLines={1} enableSearch url={url} />
+  <LazyLog extraLines={1} enableSearch url={url} caseInsensitive />
 </div>
 ```
 
+Case Sensitive Search
+
+```js
+  const url = 'https://gist.githubusercontent.com/helfi92/96d4444aa0ed46c5f9060a789d316100/raw/ba0d30a9877ea5cc23c7afcd44505dbc2bab1538/typical-live_backing.log';
+  
+  <div style={{ height: 500, width: 902 }}>
+    <LazyLog extraLines={1} enableSearch url={url} />
+  </div>
+```
+
```

Comment:
We can remove the "Case Sensitive Search" section from here. The `caseInsensitive` prop is self explanatory.